sql/catalyst/src/main/java/org/apache/spark/sql/streaming/GroupStateTimeout.java

```diff
@@ -37,7 +37,9 @@
    * `map/flatMapGroupsWithState` by calling `GroupState.setTimeoutDuration()`. See documentation
    * on `GroupState` for more details.
    */
-  public static GroupStateTimeout ProcessingTimeTimeout() { return ProcessingTimeTimeout$.MODULE$; }
+  public static GroupStateTimeout processingTimeTimeout() {
```

Comment:
CC @tdas this especially would need your attention. The name of the method, being TitleCase instead of camelCase, generates a Java style error. It has a point. We can change it because these methods are only new in 2.2, but wanted to check that it is OK? or was it on purpose and needs to be suppressed?